src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -146,11 +147,19 @@ private DashboardCommands parseDashboardCommandFromInput(String input) {
      * @param input full user input string
      * @return module code string
      */
-    private String getModuleCode(String input) {
-        // TODO  - error handling
+    private String getModuleCode(String input) throws CommandException {
         String[] words = input.split(" ");
+        if (words.length < 2) {
```

Comment:
Probably better to change the magic number to a constant.e.g NUMBER_OF_ARGUMENTS_IN_COMMAND.